openslides/core/static/js/core/site.js

```diff
@@ -573,17 +573,31 @@ angular.module('OpenSlidesApp.core.site', [
  *   instance.column='title')
  */
 .factory('osTableSort', [
-    function () {
-        var createInstance = function () {
+    '$sessionStorage',
+    function ($sessionStorage) {
+        var createInstance = function (tableName) {
             var self = {
                 column: '',
                 reverse: false,
             };
+            var existsStorageEntry = function () {
+                return $sessionStorage[tableName];
+            };
+            var storage = existsStorageEntry();
+            if (storage) {
+                self = storage;
+            }
+
+            self.existsStorageEntry = existsStorageEntry;
```

Comment:
This attribute is not used in your new code. Is it used somewhere else? If not please remove it (and then maybe do not put this logic into an extra variable and function).

Reply:
Done.